### ISSUE TITLE
AMD Vega64 CN-GPU low auto intensity.

### ISSUE THREAD

**Issue by Ryo-RU** [fireice-uk/xmr-stak#2517]
TLDR: When started V56 xmr-stak generates proper intensity in config. But V64 - not.
Algo: **CN-GPU**
Tested on 2 different devices on HiveOS and W10
[[Imgur link]](https://imgur.com/a/L8eNMkO)

**Details:**
This seem to appear on 2.10.7 (2.10.5 generates 1536 auto-intensity)
Auto intensity: 1024 (the rest is fine: worksize 16, memchunk 2, 1 thread)
Manually found that intensity 2016 is the best spot there.

```
(2023 / (16*64) * 64*16 = 1024 for Vega 64
(2023/ (16*56)) * 56*16 = 1792 for Vega 56
```
Bcs of 1,9 turning into 1 here = 64*16 gives 1024 as a result.

Both have same GPU codename: gfx900

4024/2 = threads = 2012
(2012 /  16)) * 16 = 2000
Ending with at least 2000 intensity in one thread with ws16 will be optimal with extra reserve space to prevent possible crashes. 

_Thanks to Villos1 (HiveOS team) and newtoetn from discord for feedback_

**Comment by psychocrypt** [comment]
fixed in #2518
